blocks/library/tag-cloud/editor.scss

```diff
@@ -0,0 +1,13 @@
+.gutenberg .wp-block-tag-cloud {
+    a {
+        display: inline-block;
```

Comment:
These spaces should be tabs.

>Use tabs, not spaces, to indent each property.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#structure